src/scenarios/shared/runner.py

```diff
@@ -46,20 +46,29 @@ class Runner:
     def __init__(self, traits: TestTraits):
         self.traits = traits
         self.testtype = None
+        self.sdktype = None
         setup_loggers(True)
 
     def parseargs(self):
         '''
         Parses input args to the script
         '''
         parser = ArgumentParser()
-        parser.add_argument('testtype', choices=testtypes, type=str.lower)
+        subparsers = parser.add_subparsers(title='subcommands for sdk tests', required=True, dest='testtype')
+        startupparser = subparsers.add_parser(const.STARTUP)
+        sdkparser = subparsers.add_parser(const.SDK)
+        sdkparser.add_argument('sdktype', choices=[const.CLEAN_BUILD, const.BUILD_NO_CHANGE], type=str.lower)
+
```

Comment:
This is great! I was going to need something like this for the crossgen tests anyway.